kinto_client/cli_utils.py

```diff
@@ -37,6 +37,8 @@ def set_parser_server_options(parser=None,
                               default_auth=None,
                               default_bucket=None,
                               default_collection=None,
+                              with_bucket=True,
+                              with_collection=True,
```

Comment:
I would rename "with" by "include" in order to have `include_bucket` and `include_collection`.